cirq-core/cirq/work/sampler.py

```diff
@@ -29,6 +29,7 @@
 
 if TYPE_CHECKING:
     import cirq
+    import sympy
```

Comment:
Is sympy still needed? Looks like it's not used below.

Reply:
Removed.

Reply:
Removed.